jobs/haproxy/spec

```diff
@@ -190,6 +190,9 @@ properties:
   ha_proxy.enable_health_check_http:
     description: "Optionally enable http health-check on `haproxy_ip:8080/health`. It shows `200 OK` if >0 backend servers are up. If used with ext_crt_list_timeout you should make sure that the deployment canary_watch_time and update_watch_time are configured to wait at least the number of seconds defined by ext_crt_list_timeout."
     default: false
+  ha_proxy.disable_monit_health_check_http:
+    description: "Optionally disable http health-check on `haproxy_ip:8080/health` using monit. This switch should be used to avoid testing the healthcheck before diego is up, restoring behavior as it was before the HAProxy v10.5.0. This flags works in combination with enable_health_check_http, if enable_health_check_http is false the value of this property would be ignored and no checks will be performed, if enable_health_check_http is true then setting this variable as true will disable the monit checks."
```

Comment:
Hi @jaristiz I think this could still be clearer and not all users use HAProxy with Cloud Foundry. Can you change it to:

The HAProxy health check endpoint has a healthy status if at least one backend server is responding. By default when `enable_health_check_http: true`, BOSH will consider the HAProxy VM unhealthy if the HAProxy health check returns an unhealthy status. In some cases this might not be desired, for example when deploying _HAProxy_ before deploying the backend servers. To prevent BOSH from considering the HAProxy VM unhealthy when all backend servers are unhealthy set `disable_monit_health_check_http: true`. Note that this flag is ignored unless `enable_health_check_http: true`.

Reply:
Change done.